sessionctx/variable/tidb_vars.go

```diff
@@ -892,6 +892,7 @@ const (
 	DefTiDBEnablePrepPlanCache                   = true
 	DefTiDBPrepPlanCacheSize                     = 100
 	DefTiDBPrepPlanCacheMemoryGuardRatio         = 0.1
+	DefTiDBEnableConcurrencyDDL                  = true
```

Comment:
If it is an experimental switch, shouldn't the default be false?

Reply:
it is not experimental actually, we switch off if there is something wrong with the new feature.